Dockerfile

```diff
@@ -1,24 +1,23 @@
-# Alpine provides the same multiplatform support as arch.
-FROM alpine
-
-# GOPATH AND GOBIN ON PATH
-ENV GOPATH=/go
-ENV PATH=$PATH:/go/bin
+FROM golang:1.16.2-alpine3.13
 
 # INSTALL DEPENDENCIES
-RUN apk add --no-cache go npm make git bash which protoc && \
-	mkdir /go
+RUN apk add --no-cache npm make git bash which curl protoc
+
+# INSTALL PROTOBUF LIBRARY
+ENV PROTOC_ZIP=protoc-3.13.0-linux-x86_64.zip
+RUN curl -OL https://github.com/protocolbuffers/protobuf/releases/download/v3.13.0/${PROTOC_ZIP}
+RUN unzip -o ${PROTOC_ZIP} -d /proto
+RUN cp -R /proto/include/* /usr/include/
 
 # COPY STARPORT SOURCE CODE INTO CONTAINER
 COPY . /starport
 WORKDIR /starport
 
 # INSTALL STARPORT
-RUN PATH=$PATH:/go/bin && \
-		bash scripts/install
+RUN bash scripts/install
```

Comment:
Can we use `make install` instead, it's the most up-to-date one.

Reply:
yeah, it used by the gitpod configs but that'll chance once we enter refactoring period.